.github/workflows/deploy-do-droplet-prod.yml

```
@@ -43,147 +50,33 @@ jobs:
-          APP_NAME: '${{ secrets.APP_NAME }}'
-          APP_LOGO: '${{ secrets.APP_LOGO }}'
-          APP_SIGNATURE: '${{ secrets.APP_SIGNATURE }}'
-          APP_LINK: '${{ secrets.APP_LINK }}'
-          APP_EMAIL_CONFIRMATION_URL: '${{ secrets.APP_EMAIL_CONFIRMATION_URL }}'
-          APP_MAGIC_SIGN_URL: '${{ secrets.APP_MAGIC_SIGN_URL }}'
-          COMPANY_LINK: '${{ secrets.COMPANY_LINK }}'
-          COMPANY_NAME: '${{ secrets.COMPANY_NAME }}'
+          # Include other environment variables from your workflow
+
+      - name: Copy file via SCP
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:7: Double quote to prevent globbing and word splitting

(shellcheck)